src/views/v2/rsvp/actions/success.php

```diff
@@ -39,9 +35,26 @@ class="tribe-common-form-control-toggle__input tribe-tickets__rsvp-actions-succe
 			class="tribe-common-form-control-toggle__label tribe-tickets__rsvp-actions-success-going-toggle-label"
 			for="<?php echo esc_attr( $toggle_id ); ?>"
 		>
-			<?php esc_html_e( 'Show me on public attendee list', 'event-tickets' ); ?>
+			<span
+				data-js="tribe-tickets-tooltip"
+				data-tooltip-content="#tribe-tickets-tooltip-content-<?php echo esc_attr( $rsvp->ID ); ?>"
+				aria-describedby="tribe-tickets-tooltip-content-<?php echo esc_attr( $rsvp->ID ); ?>"
+			><?php
+				echo wp_kses_post(
+					sprintf(
+						// Translators: 1: opening span. 2: Closing span.
+						_x(
+							'Show me on public %1$sattendee list%2$s',
+							'Toggle for RSVP attendee list.',
+							'event-tickets'
+						),
+						'<span class="tribe-tickets__rsvp-actions-success-going-toggle-label-underline">',
+						'</span>'
+					)
+				);
+			?></span>
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected 4 tabs, found 3
* :no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 9a59ebd3c1fe8dfcd27e18bac1bed7f3</sub>